lib/chef/user.rb

```diff
@@ -144,12 +287,17 @@ class << self
     end
 
     def self.list(inflate=false)
-      response = Chef::REST.new(Chef::Config[:chef_server_url]).get_rest('users')
-      users = if response.is_a?(Array)
```

Comment:
Yeah, Jay gave a great suggestion on how to handle `knife user create` for CS 11 that I will implement. Basically, if you only pass the one arg (username) it will just try against `Chef::OscUser` with a warning that the command has been moved to `knife osc_user create`.